File: c/1074.c
#include <stdio.h>
#include <string.h>

int main()
{
    int i, e, x;
    char str[80];
    scanf("%d", &e);

    for (i = 0; i < e; i++)
    {
        scanf("%d", &x);
        str[0] = '\0'; 

        if (x == 0)
        {
            printf("NULL\n");
        }
        else
        {
            if (x % 2 == 0)
            {
                strcat(str, "EVEN");
            }
            else
            {
                strcat(str, "ODD");
            }

            if (x > 0)
            {
                strcat(str, " POSITIVE");
            }
            else
            {
                strcat(str, " NEGATIVE");
            }
            printf("%s\n", str);
        }
    }
    return 0;
}

File: php/PROBLEMS/1074.php
<?php
    $e = intval(fgets(STDIN));
    
    for($i=0;$i<$e;$i++){
        $x = intval(fgets(STDIN));
        $str = '';
        if ($x == 0)
        {
            printf("NULL\n");
        }
        else
        {
            if ($x % 2 == 0)
            {
                $str.="EVEN";
            }
            else
            {
                $str.="ODD";
            }

            if ($x > 0)
            {
                $str.=" POSITIVE";
            }
            else
            {
                $str.=" NEGATIVE";
            }
            printf("%s\n", $str);
        }
    }
?>